libs/application/template-api-modules/src/lib/modules/templates/accident-notification/accodent-notification-v2.utils.ts

```
@@ -0,0 +1,379 @@
+      postcode: homeAccident.postalCode ?? '',
+      comment: homeAccident.moreDetails ?? '',
+    }
+  )
+}
+
+const getAtWork = (answers: AccidentNotificationAnswers) => {
+  const workMachine = getValueViaPath(answers, 'workMachine') as WorkMachineV2
+
+  return workMachine && workMachine.descriptionOfMachine
+    ? {
+        machineDescription: workMachine?.descriptionOfMachine,
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)